### ISSUE TITLE
Development Plan

### ISSUE THREAD

**Issue by ftshijt** [SJTMusicTeam/SVS_system#19]
Refine current code

- [x] remove abundant files
- [x] modulization (e.g., mel layer, mfcc, fbank,, vocoder (world)) 
- [x] normalization (norm, collect stats)
- [x] make "egs", make separate data running scripts (\w different configuration and data_prep process)
- [x] refine README.md

- [x] autoregressive module

**Comment by ghost** [comment]
@ftshijt Hi, great work can you please add example audio and a google colab for inference?

**Comment by ftshijt** [comment]
> @ftshijt Hi, great work can you please add example audio and a google colab for inference?

Thanks for your interest. We planned to have some samples available. Google colab would definitely be a good choice for that.

**Comment by ghost** [comment]
@ftshijt is this the repo for https://arxiv.org/abs/2010.12024

**Comment by ftshijt** [comment]
> @ftshijt is this the repo for https://arxiv.org/abs/2010.12024

Yes, this is the repo for the paper. We are currently interested in making it a more comprehensive toolkit like [nnsvs](https://github.com/r9y9/nnsvs) or [espnet](https://github.com/espnet/espnet).
